101-synapse-poc/nestedtemplates/logicapproleassignments.json

```diff
@@ -0,0 +1,45 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "logicAppPauseName": {
+            "type": "string"
+        },
+        "logicAppResumeName": {
+            "type": "string"
+        },
+        "pauseGuid": {
+            "type": "string",
+            "defaultValue": "[newGuid()]"
```

Comment:
remove defaultValue since this must be passed through - also, this needs to be idempotent otherwise it will fail if redeployed.